arlo-client/src/components/Audit/RoundManagement/CreateAuditBoards.tsx

```diff
@@ -2,73 +2,44 @@ import React from 'react'
 import { H4 } from '@blueprintjs/core'
 import { Formik, Field } from 'formik'
 import styled from 'styled-components'
-import { toast } from 'react-toastify'
-import { IAuditBoard, IErrorResponse } from '../../../types'
 import { generateOptions } from '../../Atoms/Form/_helpers'
 import FormButton from '../../Atoms/Form/FormButton'
 import Select from '../../Atoms/Form/Select'
-import { api, checkAndToast } from '../../utilities'
 
 const Wrapper = styled.div`
   margin: 20px 0;
 `
 
-const CreateAuditBoards = ({
-  auditBoards,
-  electionId,
-  jurisdictionId,
-  roundId,
-  getAuditBoards,
-}: {
-  auditBoards: IAuditBoard[]
-  electionId: string
-  jurisdictionId: string
-  roundId: string
-  getAuditBoards: () => void
-}) => {
-  const submit = async ({
-    auditBoards: auditBoardsCount,
-  }: {
-    auditBoards: number
-  }) => {
-    try {
-      const body = JSON.stringify(
-        [...Array(auditBoardsCount).keys()].map(i => ({
-          name: `Audit Board #${i}`,
-        }))
-      )
-      const response: IErrorResponse = await api(
-        `/election/${electionId}/jurisdiction/${jurisdictionId}/round/${roundId}/audit-board`,
-        {
-          method: 'POST',
-          headers: {
-            'Content-Type': 'application/json',
-          },
-          body,
-        }
-      )
-      // checkAndToast left here for consistency and reference but not tested since it's vestigial
-      /* istanbul ignore next */
-      if (checkAndToast(response)) return
-      getAuditBoards()
-    } catch (err) /* istanbul ignore next */ {
-      // TEST TODO
-      toast.error(err.message)
-    }
+interface IValues {
+  numAuditBoards: number
+}
+
+interface IProps {
+  createAuditBoards: (auditBoards: { name: string }[]) => Promise<boolean>
+  disabled: boolean
+}
+
+const CreateAuditBoards = ({ createAuditBoards, disabled }: IProps) => {
+  const submit = async ({ numAuditBoards }: IValues) => {
+    const boards = [...Array(numAuditBoards).keys()].map(i => ({
+      name: `Audit Board #${i}`,
```

Comment:
@jonahkagan I think this is a bug I found earlier but didn't get around to fixing. This should be `i + 1` so it's more user-friendly (not zero-indexed).

Reply:
oooh nice catch thank you